src/lib/tests/createMockNetworkLayer/__tests__/createMockNetworkLayer-tests.tsx

```diff
@@ -30,29 +30,38 @@ describe("createMockNetworkLayer", () => {
   describe("preserves the upstream behaviour", () => {
     it("returns the data if present", async () => {
       const data = await fetchQueryWithResolvers({
```

Comment:
Rename these either now or in a next PR to be about “data” instead of “resolvers”.

Reply:
Updating this, this should also be updated in Reaction, as this was copied from there.